var/spack/repos/builtin/packages/cp2k/package.py

```diff
@@ -207,8 +215,8 @@ class Cp2k(MakefilePackage, CMakePackage, CudaPackage, ROCmPackage):
 
     depends_on("blas")
     depends_on("lapack")
-    depends_on("fftw-api@3")
-
+    depends_on("fftw-api@3", when="+fftw3")
```

Comment:
The `fftw-api` is a virtual dependency. I think it should be always enabled, not just when one wants to use `fftw3` as the FFT library.

Reply:
OK, I will fix it asap